src/models/goal.js

```diff
@@ -61,6 +61,7 @@ export default (sequelize, DataTypes) => {
       });
       Goal.hasMany(models.SimScoreGoalCache, { foreignKey: 'goal1', as: 'scoreOne' });
       Goal.hasMany(models.SimScoreGoalCache, { foreignKey: 'goal2', as: 'scoreTwo' });
+      Goal.hasMany(models.GoalStatusChange, { foreignKey: 'goalId', as: 'statusChanges' });
```

Comment:
Just a preference, but I prefer that all association are placed in the file causing the association. The file with the reference, not the model being referenced. It prevents some potential race conditions and keeps each model a little cleaner.